test/scalarstats.jl

```diff
@@ -71,7 +71,7 @@ z2 = [8. 2. 3. 1.; 24. 10. -1. -1.; 20. 12. 1. -2.]
 
 ##### Dispersion
 
-@test span([3, 4, 5, 6, 2]) == (2:6)
+@test StatsBase.span([3, 4, 5, 6, 2]) == (2:6)
```

Comment:
@ararslan I needed to do this because previously I was using Distributions.jl to generate better random probability distributions. Since I removed the dependency, it's no longer necessary (so I can remove it?), but it's worth noting that StatsBase.jl and Distributions.jl provide functions called span() than do different things.